sdk/js/src/token_bridge/transfer.ts

```diff
@@ -95,6 +95,33 @@ export async function transferFromEth(
   return receipt;
 }
 
+export async function transferFromEthWithPayload(
```

Comment:
why is this another function but the solana and terra ones are shared? isn't algorand shared as well?